insights/parsers/hammer_compute_resource_list.py

```diff
@@ -0,0 +1,56 @@
+"""
+HammerComputeResourceList - command ``hammer --interactive 0 --output json compute-resource list``
+===================================================
+
+This parser reads the compute resource list of a Satellite server using hammer in JSON format.
+It relies on the root user running the command and hammer to authenticate based on the hammer configuration file.
+If the command is unable to authenticate then no compute resources will be shown and an erro flag will be recorded in the parser
+
+Sample output from the ``hammer --interactive 0 --output json compute-resource list`` command::
+
+    [
+  {
+    "Id": 1,
+    "Name": "kvm-server",
+    "Provider": "Libvirt"
+  },
+  {
+    "Id": 4,
+    "Name": "VMWARE",
+    "Provider": "VMware"
+  },
+  {
+    "Id": 3,
+    "Name": "vmware67",
+    "Provider": "VMware"
+  }
+]
+
+Examples:
+
+    >>> type(hammer_compute_resources_list)
+    <class 'insights.parsers.hammer_compute_resource_list.HammerComputeResourceList'>
```

Comment:
Would you please add a bit more to the `Examples`, e.g. how many resources are listed (the length of the list, `len(hammer_compute_resources_list)`, or how to access a specific value of a resource?